styleguide/derek/incubation/homepage.ejs

```diff
@@ -1,146 +1,64 @@
 <%- partial("../pattern-components/headers/header-refresh-homepage.ejs") %>
 
-<div class="container half-padding hostingOptions">
-	<div class="row half-padding-full">
-		<div class="col-md-11 center-block">
-			<h1 class="text-center no-margin no-padding" id="homepage-lead-top">Rackspace is the largest managed cloud provider</h1>
-			<h2 class="text-center no-margin no-padding" id="homepage-lead-bottom">offering you expertise across the world's leading clouds</h2>
+<div class="rsHomepage-announcement">
+	<div class="rsHomepage-leftStripContainer">
+		<div class="rsHomepage-annLeftBar">
+			<div class="rsHomepage-titleContainer">
+				<h2 class="rsHomepage-annLeftBar-header">TRANSFORM THE WAY YOU DO BUSINESS</h2>
+				<h3 class="rsHomepage-annLeftBar-subHeader">Offering expertise and support for your IT transformation</h3>
+			</div>
 		</div>
+		<div class="rsHomepage-annRight-iconBar"></div>
 	</div>
-	<div class="row row-eq-height">
```

Comment:
@WidgetsBurritos we discussed this. I had to remove these sections in order to see the homepage as it will be. No styling is removed in this PR. Just some of this random leftover junk.